test/test_workflow_Run2016B.py

```diff
@@ -63,22 +66,19 @@
 #         os.remove(fname)
 
 # check drifts and create IOV
-#filename = 'BS_result_PiltBeam2021_byIOV/beamspot_Express_PilotBeam2021_.txt'
 count = 0
 for irun, ibs in allBS.items():
     if irun not in specialRuns:
-      pairs = splitByDrift(ibs, slopes = True)    
+      pairs = splitByDrift(ibs, slopes = True, maxLumi = 20)    
     else:
-      pairs = splitByDrift(ibs, slopes = True, maxLumi = 1)    
+      pairs = splitByDrift(ibs, slopes = True)    
```

Comment:
same here, for special runs (i.e. usually VdM) we want to have it every LS, so please change it back.